src/js/controller/controller.js

```diff
@@ -295,6 +302,13 @@ define([
                 }
             }
 
+            // Should only attempt to preload if the player is viewable.
+            // Otherwise, it should try to preload the first player on the page,
+            // which is the player that has a uniqueId of 1
+            function shouldPreload(preloaded, viewable) {
+                return !preloaded && (viewable === 1 || _api.uniqueId === 1);
```

Comment:
`api.uniqueId === 1` is the first player instance created with `jwplayer().setup()`, but there could be cases where old players are removed. So what we really want is to know if `_api === _instances[0]` in global-api.js. An easy what to do this is `_api === jwplayer(0)` or `selectPlayer(0)`.